[0m13:16:30.989941 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x793f91f03fa0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x793f904df760>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x793f904df700>]}


============================== 13:16:30.992633 | 15a83bcd-1401-4d30-b347-1a0e83816156 ==============================
[0m13:16:30.992633 [info ] [MainThread]: Running with dbt=1.8.9
[0m13:16:30.993251 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/vscode/.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt init', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m13:16:31.006875 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object.. Dictionary: {'dir': PosixPath('/home/vscode/.dbt')}
[0m13:16:31.007671 [info ] [MainThread]: Creating dbt configuration folder at 
[0m13:23:46.140532 [debug] [MainThread]: Starter project path: /home/vscode/.local/lib/python3.10/site-packages/dbt/include/starter_project
[0m13:23:46.147457 [info ] [MainThread]: 
Your new dbt project "coffee_shop_louissa" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m13:23:46.148348 [info ] [MainThread]: Setting up your profile.
[0m13:25:34.855390 [info ] [MainThread]: Profile coffee_shop_louissa written to /home/vscode/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m13:25:34.856997 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 543.9114, "process_in_blocks": "736", "process_kernel_time": 0.139591, "process_mem_max_rss": "201244", "process_out_blocks": "88", "process_user_time": 1.570181}
[0m13:25:34.857701 [debug] [MainThread]: Command `dbt init` succeeded at 13:25:34.857579 after 543.91 seconds
[0m13:25:34.858141 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x793f91f03fa0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x793f904df730>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x793f904dff70>]}
[0m13:25:34.858570 [debug] [MainThread]: Flushing usage events
[0m11:31:47.730473 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b961a987e20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b9618fac7c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b9618fac850>]}


============================== 11:31:47.735737 | b55e28ab-caf5-49ea-9698-21c2ea417e9d ==============================
[0m11:31:47.735737 [info ] [MainThread]: Running with dbt=1.8.9
[0m11:31:47.736566 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/vscode/.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m11:31:47.737343 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/ae-club-louissa/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:31:47.738834 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": false, "command_wall_clock_time": 0.053731598, "process_in_blocks": "68336", "process_kernel_time": 0.149653, "process_mem_max_rss": "85316", "process_out_blocks": "24", "process_user_time": 1.136556}
[0m11:31:47.739535 [debug] [MainThread]: Command `dbt run` failed at 11:31:47.739427 after 0.05 seconds
[0m11:31:47.739952 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b961a987e20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b9618fac7c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b9618fac100>]}
[0m11:31:47.740357 [debug] [MainThread]: Flushing usage events
[0m16:00:21.731223 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x79ce7073bdc0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x79ce6ed18880>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x79ce6ed188e0>]}


============================== 16:00:21.735120 | 97537919-1c95-447b-a102-b1d3dc46cded ==============================
[0m16:00:21.735120 [info ] [MainThread]: Running with dbt=1.8.9
[0m16:00:21.735843 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/vscode/.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt run', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:00:21.736402 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/ae-club-louissa/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m16:00:21.737700 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": false, "command_wall_clock_time": 0.05216583, "process_in_blocks": "58624", "process_kernel_time": 0.132673, "process_mem_max_rss": "85604", "process_out_blocks": "8", "process_user_time": 1.097567}
[0m16:00:21.738336 [debug] [MainThread]: Command `dbt run` failed at 16:00:21.738238 after 0.05 seconds
[0m16:00:21.738726 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x79ce7073bdc0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x79ce6ed18880>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x79ce6ed18220>]}
[0m16:00:21.739137 [debug] [MainThread]: Flushing usage events
[0m16:11:40.401180 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x6ffdc26afe50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x6ffdc2a7b700>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x6ffdc2a7b6a0>]}


============================== 16:11:40.403775 | bf1ec0ba-e0fd-4e71-acdc-cefa7869364b ==============================
[0m16:11:40.403775 [info ] [MainThread]: Running with dbt=1.8.9
[0m16:11:40.404481 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': '/home/vscode/.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'invocation_command': 'dbt init demo-dbt-project', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:12:18.675610 [debug] [MainThread]: Starter project path: /home/vscode/.local/lib/python3.10/site-packages/dbt/include/starter_project
[0m16:12:18.682838 [info ] [MainThread]: 
Your new dbt project "demo_dbt_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:12:18.683444 [info ] [MainThread]: Setting up your profile.
[0m11:57:28.375834 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7adee1a67df0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7adee1e3b970>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7adee1e3b910>]}


============================== 11:57:28.380958 | 0d995516-3654-4b2d-99c4-ccd7c1c5fd81 ==============================
[0m11:57:28.380958 [info ] [MainThread]: Running with dbt=1.8.9
[0m11:57:28.383400 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': '/home/vscode/.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt docs generate', 'send_anonymous_usage_stats': 'True'}
[0m11:57:28.384508 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/ae-club-louissa/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:57:28.385979 [debug] [MainThread]: Resource report: {"command_name": "generate", "command_success": false, "command_wall_clock_time": 0.055633802, "process_in_blocks": "68336", "process_kernel_time": 0.112751, "process_mem_max_rss": "85384", "process_out_blocks": "8", "process_user_time": 1.12349}
[0m11:57:28.387098 [debug] [MainThread]: Command `dbt docs generate` failed at 11:57:28.386941 after 0.06 seconds
[0m11:57:28.387705 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7adee1a67df0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7adee1e3b970>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7adee1e3b640>]}
[0m11:57:28.388334 [debug] [MainThread]: Flushing usage events
[0m13:15:24.084797 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74d31673fe80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74d314d27730>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74d314d276d0>]}


============================== 13:15:24.090850 | 6286a742-5f70-4ebb-9045-da8090d37a36 ==============================
[0m13:15:24.090850 [info ] [MainThread]: Running with dbt=1.8.9
[0m13:15:24.092002 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/home/vscode/.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt init demo_dbt_project', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m13:15:24.108214 [info ] [MainThread]: A project called demo_dbt_project already exists here.
[0m13:15:24.109998 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 0.07691064, "process_in_blocks": "68712", "process_kernel_time": 0.161277, "process_mem_max_rss": "85848", "process_out_blocks": "8", "process_user_time": 1.266616}
[0m13:15:24.110796 [debug] [MainThread]: Command `dbt init` succeeded at 13:15:24.110656 after 0.08 seconds
[0m13:15:24.111235 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74d31673fe80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74d314cfb7f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74d314d240d0>]}
[0m13:15:24.111683 [debug] [MainThread]: Flushing usage events
[0m14:09:22.830117 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1e3e0beb0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1e2443a30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1e24439d0>]}


============================== 14:09:22.833153 | 037c7c28-51c7-4e1b-a596-d0c4fc304451 ==============================
[0m14:09:22.833153 [info ] [MainThread]: Running with dbt=1.8.9
[0m14:09:22.833929 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/home/vscode/.dbt', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt docs generate', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m14:09:22.834490 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/ae-club-louissa/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m14:09:22.835471 [debug] [MainThread]: Resource report: {"command_name": "generate", "command_success": false, "command_wall_clock_time": 0.051189493, "process_in_blocks": "0", "process_kernel_time": 0.063752, "process_mem_max_rss": "85540", "process_out_blocks": "16", "process_user_time": 1.155512}
[0m14:09:22.836230 [debug] [MainThread]: Command `dbt docs generate` failed at 14:09:22.836117 after 0.05 seconds
[0m14:09:22.836642 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1e3e0beb0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1e2443a30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa1e2443640>]}
[0m14:09:22.837053 [debug] [MainThread]: Flushing usage events
[0m14:10:04.126305 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x72e3200efdc0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x72e31e723820>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x72e31e7237c0>]}


============================== 14:10:04.129322 | 9174c54b-0c29-4fc4-82d5-148a6cf41d2c ==============================
[0m14:10:04.129322 [info ] [MainThread]: Running with dbt=1.8.9
[0m14:10:04.130090 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/vscode/.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt docs serve', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m14:10:04.130650 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/ae-club-louissa/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m14:10:04.131543 [debug] [MainThread]: Resource report: {"command_name": "serve", "command_success": false, "command_wall_clock_time": 0.051223613, "process_in_blocks": "0", "process_kernel_time": 0.055936, "process_mem_max_rss": "85836", "process_out_blocks": "8", "process_user_time": 1.106582}
[0m14:10:04.132262 [debug] [MainThread]: Command `dbt docs serve` failed at 14:10:04.132153 after 0.05 seconds
[0m14:10:04.132649 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x72e3200efdc0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x72e31e723820>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x72e31e723e80>]}
[0m14:10:04.133045 [debug] [MainThread]: Flushing usage events
